packages/big-design/src/components/Table/Search/Search.tsx

```diff
@@ -0,0 +1,37 @@
+import { SearchIcon } from '@bigcommerce/big-design-icons';
+import React, { FormEvent } from 'react';
+
+import { Button } from '../../Button';
+import { Form } from '../../Form';
+import { Input } from '../../Input';
+import { TableSearch } from '../types';
+
+import { StyledFlex, StyledInput } from './styled';
+
+export const Search: React.FC<TableSearch> = ({ value, onSearchChange, onSearchSubmit }) => {
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    onSearchSubmit?.(event);
+  };
+
+  return (
+    <Form onSubmit={handleSubmit} fullWidth={true}>
+      <StyledFlex alignItems="center" flexDirection="row" justifyContent="stretch">
+        <StyledInput>
+          <Input
+            placeholder="Search"
+            type="text"
+            value={value}
+            onChange={onSearchChange}
+            iconLeft={<SearchIcon color="secondary" />}
```

Comment:
From the designs, looks like this should be `secondary50` instead.
```suggestion
            iconLeft={<SearchIcon color="secondary50" />}
```

Reply:
You are right. Thank you!